dealroom_firestore_connector/__init__.py

```diff
@@ -622,25 +750,41 @@ def _validate_new_people_doc_payload(payload: dict):
     """Validate the required fields in the payload when creating a new document in the people collection"""
 
     if "dealroom_id" not in payload:
-        raise KeyError("'dealroom_id' must be present payload")
-
-    _validate_dealroomid(payload["dealroom_id"])
+        if "dealroom_uuid" not in payload:
+            raise KeyError(
+                "at least one of 'dealroom_id', 'dealroom_uuid' must be present in payload"
+            )
+        else:
+            _validate_dealroom_uuid(payload["dealroom_uuid"])
+    else:
+        _validate_dealroom_id(payload["dealroom_id"])
+        if "dealroom_uuid" in payload:
+            _validate_dealroom_uuid(payload["dealroom_uuid"])
```

Comment:
```
if "dealroom_id" not in payload and "dealroom_uuid" not in payload:
    raise KeyError(
                "at least one of 'dealroom_id', 'dealroom_uuid' must be present in payload"
            )

if "dealroom_id" in payload:
    _validate_dealroom_id(payload["dealroom_id"])
if "dealroom_uuid" in payload:
    _validate_dealroom_uuid(payload["dealroom_uuid"])
```

Reply:
Is it the same? 🤔  my eyes are crossing 🤦

Reply:
Changed in https://github.com/dealroom/data-firestore-connector/pull/31/commits/70bebf4fc2e28719d6efdc98bbc2cdbde62d501b